src/Tpay/Factory/CreateRedirectBasedPaymentPayloadFactory.php

```diff
@@ -35,7 +37,10 @@ public function createFrom(PaymentInterface $payment, string $notifyUrl, string
 
         return [
             'amount' => number_format($amount / 100, 2, thousands_separator: ''),
-            'description' => sprintf('zamówienie #%s', $order->getNumber()), // TODO: Introduce translations
+            'description' => $this->translator->trans(
+                'commerce_weavers_sylius_tpay.shop.api.description.order',
```

Comment:
```suggestion
                'commerce_weavers_sylius_tpay.tpay.transaction_description',
```
This is a description we send to the Tpay in transaction. The translatable key I proposed seems more valid to me, as this is not related to shop and api.

Reply:
It might also be `common`